embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/comms/api/Endpoint.kt

```diff
@@ -0,0 +1,14 @@
+package io.embrace.android.embracesdk.internal.comms.api
+
+public enum class Endpoint(
+    public val path: String,
+    public val version: String
+) {
+    EVENTS("events", "v1"),
+    LOGGING("logging", "v1"),
```

Comment:
Many of these can be deleted because they're no longer used?

Reply:
I'll address this in a separate PR because while some of the endpoints might not be used by the SDK any more, there is a non-trivial amount of code associated with them